src/config.rs

```diff
@@ -81,6 +82,7 @@ mod cmd {
 
 mod arg {
   pub(crate) const ARGUMENTS: &str = "ARGUMENTS";
+  pub(crate) const CHECK_FMT: &str = "CHECK";
```

Comment:
The const name should match the value, so:
```suggestion
  pub(crate) const CHECK: &str = "CHECK";
```